pkg/etcd/etcd.go

```diff
@@ -219,6 +219,10 @@ func (e *ETCD) Start(ctx context.Context, clientAccessInfo *clientaccess.Info) e
 	go e.manageLearners(ctx)
 
 	if existingCluster {
+		info, err := os.Stat(etcdDBDir(e.config))
+		if info.Mode().Perm() != 0700 {
+			os.Chmod(info.Name(), 0700)
+		}
```

Comment:
err return appears to be unused - need either return err if non-nil, or check for nil error before looking at info.
```suggestion
		if info, err := os.Stat(etcdDBDir(e.config)); err == nil && info.Mode().Perm() != 0700 {
			os.Chmod(info.Name(), 0700)
		}
```